docs/release/docker_image_support_matrix.md

```diff
@@ -61,7 +61,7 @@ The software support matrices for ROCm container releases is listed.
 
 * [ROCm5.6](https://repo.radeon.com/rocm/apt/latest/)
 * [Python 3.9](https://www.python.org/downloads/release/python-390/)
-* [tensorflow-rocm 2.13.0]()
+* [tensorflow-rocm 2.13.0](#tensorflow-rocm 2.13.0)
```

Comment:
This renders oddly in the preview (https://advanced-micro-devices-demo--2180.com.readthedocs.build/en/2180/release/docker_image_support_matrix.html). Maybe we should leave this as plain text until we have a correct link.

eg:
`* tensorflow-rocm 2.13.0`

Reply:
Removed the link!